R/modify-model-field.R

```diff
@@ -41,7 +41,7 @@ modify_model_field <- function(.mod, .field, .value, .append = TRUE, .unique = T
 
 #' Add tags to a model object and corresponding YAML
 #' @param .mod The `bbi_{.model_type}_model` object to modify
-#' @param .tags Character scaler or vector of tags to add
+#' @param .tags Character vector of tags to add
```

Comment:
This description will not appear in the documentation because we also have `@rdname modify_model_field`, so the description for `.tags` in the function of the same name will be used. If this description needs to be different, there are a few strategies:

1. Document in separate files (so, change or remove the `@rdname` tag).

2. Generalize the description for `.tags` in `modify_model_field()` to incorporate behavior relevant to any function with that argument that is documented in the same file.

3. Change the argument name, which will result in creating a new argument with the accompanying description.

Note also that if one function takes an argument whose description is the same as found in another function, we can use [`@inheritParams`](http://r-pkgs.had.co.nz/man.html#dry2) to use the documentation from that function, e.g., `@inheritParams modify_model_field`. If the functions share an argument of the same name, but its description differs, then it can be handled as above, or a new `@param` description added, which will override anything inherited from `@inheritParams`.

Reply:
Yes... this is a good call. I see several places where I used `rdname` haphazardly and created some sloppy docs. I'll correct that.

The 0.7.0 release is kinda being thought of as the "first public release" (because it's the first one most scientists will see) so I think it is worth spending the time to get some of this stuff right. Thanks for the fine-grained review.